site/src/Librecores/ProjectRepoBundle/Entity/OrganizationMember.php

```diff
@@ -0,0 +1,204 @@
+<?php
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * An Organization Member
+ *
+ * An organization member is a mapping between a User and an Organization he belongs.
+ *
+ * @ORM\Table("OrganizationMember")
+ * @ORM\Entity
+ * @ORM\HasLifecycleCallbacks
+ */
+class OrganizationMember
+{
+    const PERMISSIONS_REQUEST = 'REQUEST';
+    const PERMISSIONS_DENY    = 'DENY';
+    const PERMISSIONS_SUPPORT = 'SUPPORT';
+    const PERMISSIONS_MEMBER  = 'MEMBER';
+    const PERMISSIONS_ADMIN   = 'ADMIN';
```

Comment:
I mostly agree and was set to remove the 'S' but I decided not to since these might actually represent a set of permissions not only a single fine-grained one. Currently I think they are very simple and justify either meaning so I can't decide. Maybe rename to ROLE_ but this mentally classes with the symfony security roles. Uhh... why everything must be so complicated! Let's stick with the basic rule that until something changes what it does it's best to name it accordingly to what it does NOW. So I agree let's lose the 'S'.